tests/fast-integration/test/1-commerce-mock.js

```diff
@@ -61,12 +69,26 @@ function commerceMockTests() {
       await sendEventAndCheckResponse();
     });
 
+    // renew certificate
+    it("CommerceMock should renew it's certificate", async function () {
+      await renewCommerceMockCertificate();
+    });
+    // call lambda and succeed
```

Comment:
Maybe instead of commenting on the tests, we can enhance their names? For example `the event should trigger the function after the certificate renewal`